content-src/components/Spotlight/Spotlight.js

```diff
@@ -98,14 +123,17 @@ const Spotlight = React.createClass({
       page: "NEW_TAB"
     };
   },
-  onClickFactory(index) {
+  onClickFactory(index, site) {
     return () => {
       this.props.dispatch(actions.NotifyEvent({
         event: "CLICK",
         page: this.props.page,
         source: "FEATURED",
         action_position: index
       }));
+      if (site.recommended) {
+        this.props.dispatch(actions.NotifyBlockRecommendation(site.url));
```

Comment:
the idea is that if you click on the recommendation, you shouldn't be shown that recommendation again, so we just put it on the "block list" of recommendations. the addon picks a new one that isn't blocked.
i guess blocked is kind of a strong word